src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -485,3 +485,69 @@ public static DisplayInformation GetForMonitor(IntPtr monitor)
     }
 #endif
 }
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+#if UAC_VERSION_1
+#if NET
+    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.10240.0")]
+#endif
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class DataTransferManagerInterop
+    {
+        private static readonly global::System.Guid IDataTransferManagerInterop_IID = new global::System.Guid("3A3DCD6C-3EAB-43DC-BCDE-45671CE800C8");
+
+        private static readonly global::System.Guid riid = new global::System.Guid("A5CAEE9B-8708-49D1-8D36-67D25A8DA00C");
+
+        private static global::WinRT.FactoryObjectReference<global::WinRT.IObjectReference> factoryObjectReference;
+
+        static DataTransferManagerInterop()
+        {
+            // typeName = typeof(Windows.ApplicationModel.DataTransfer.DataTransferManager).FullName
+            factoryObjectReference = global::WinRT.ActivationFactory.Get<global::WinRT.IObjectReference>("Windows.ApplicationModel.DataTransfer.DataTransferManager", IDataTransferManagerInterop_IID);
+        }
+
+        public static DataTransferManager GetForWindow(global::System.IntPtr appWindow)
+        {
+            return IIDataTransferManagerInteropMethods.GetForWindow(factoryObjectReference, appWindow, riid);
+        }
+
+        public static void ShowShareUIForWindow(global::System.IntPtr appWindow)
+        {
+            IIDataTransferManagerInteropMethods.ShowShowShareUIForWindow(factoryObjectReference, appWindow);
+        }
+    }
+#endif
+
+    internal static class IIDataTransferManagerInteropMethods
+    {
+        internal static unsafe DataTransferManager GetForWindow(global::WinRT.IObjectReference _obj, global::System.IntPtr appWindow, in global::System.Guid riid)
+        {
+            global::System.IntPtr thisPtr = _obj.ThisPtr;
+            global::System.IntPtr ptr = new global::System.IntPtr();
+
+            try
+            {
+                // IDataTransferManagerInterop inherits IUnknown (3 functions) and provides GetForWindow giving a total of 5 functions
+                global::WinRT.ExceptionHelpers.ThrowExceptionForHR((*(delegate* unmanaged[Stdcall]<global::System.IntPtr, global::System.IntPtr, global::System.Guid, out global::System.IntPtr, int>**)thisPtr)[3](thisPtr, appWindow, riid, out ptr));

```

Comment:
```suggestion
                global::WinRT.ExceptionHelpers.ThrowExceptionForHR((*(delegate* unmanaged[Stdcall]<global::System.IntPtr, global::System.IntPtr, global::System.Guid, global::System.IntPtr*, int>**)thisPtr)[3](thisPtr, appWindow, riid, &ptr));

```

Reply:
This addresses 2 things.  The first is it removes `out` to make the function pointer delegate call blittable.  We fixed most of them but apparently we missed one above.  Will look at fixing at that in another PR, but we should make sure this is blittable.  This other is the GUID parameter is `REFIID` which is a pointer to a GUID as is in the other generated ones which pass the IID as a parameter.